redis/src/test/scala/zio/redis/StringsSpec.scala

```diff
@@ -1435,6 +1437,92 @@ trait StringsSpec extends BaseSpec {
             len <- strLen(key).either
           } yield assert(len)(isLeft(isSubtype[WrongType](anything)))
         }
+      ),
+      suite("getEx")(
+        testM("value exists after removing ttl") {
+          for {
+            key    <- uuid
+            value  <- uuid
+            _      <- pSetEx(key, 10.millis, value)
+            exists <- getEx(key, true)
+            _      <- ZIO.sleep(20.millis)
+            res    <- get(key)
+          } yield assert(res.isDefined)(equalTo(true)) && assert(exists)(equalTo(Some(value)))
+        } @@ eventually,
+        testM("not found value when set seconds ttl") {
+          for {
+            key    <- uuid
+            value  <- uuid
+            _      <- set(key, value)
+            exists <- getEx(key, Expire.SetExpireSeconds, 1.second)
+            _      <- ZIO.sleep(1020.millis)
```

Comment:
I would reduce the sleep time to something around 10millis

Reply:
The problem here is that the minimum expiration time is one second.

Reply:
Ok I see, thanks for clarifying